backend/src/main/java/atlantbh/restaurants/controllers/MenuController.java

```diff
@@ -0,0 +1,15 @@
+package atlantbh.restaurants.controllers;
+
+import atlantbh.restaurants.models.Menu;
+import atlantbh.restaurants.services.MenuService;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class MenuController extends BaseController<Menu, MenuService> {
+
+    public ResponseEntity menusByRestuarant(@PathVariable Long restaurant) {
```

Comment:
this can also be filter where query param would be restaurantId={id}, than you can use filter api for other properties also (no matter if there is only restaurant property in menu service, filter should be pattern, we made it base classess)

Reply:
Fixed in 0cd4debab433260ab6a8267aa0b72b31db581edb